src/components/Dropzone.svelte

```diff
@@ -182,6 +182,9 @@
     dragTargetsRef = [];
 
     if (isEvtWithFiles(event)) {
+      dispatch("startprocess", {

```

Comment:
not sure this helps. But I am as long as it's not breaking. Can we have better name. May be `startDrag` instead?

Reply:
Thank you for your nice proposition. 

I had a similar problem because when users dropped hundreds of files at once, it took time to process and caused confusion for users.

However, this will be dispatch immediately after the file is dropped. That means the drag operation has already ended, but the `startDrag` flag may cause confusion.. How about `fileDropped` instead?

Reply:
hmm good point. `fileDropped` works for me.

Reply:
OK. I changed the event name and added it to the Readme file. However, other event names don't seem to use camelCase, so I didn't use camelCase either.